OracleWebLogic/dockerfiles/12.2.1.4/Dockerfile.generic

```diff
@@ -47,8 +47,9 @@ ENV ORACLE_HOME=/u01/oracle \
 # Adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
 # ------------------------------------------------------------
 RUN mkdir -p /u01 && \
-    chmod a+xr /u01 && \
-    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle
+    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle && \
+    chown oracle:root -R /u01 && \
+    chmod -R 775 /u01
```

Comment:
remove the "-R".  There is nothing in this folder.  You just created it in this same RUN command.  Also, you can remove the -p on the mkdir.  The -p option says to create intermediate folders.  With "/u01", there are no intermediate folders, only the first/only one u01.